src/gmp/models/ociImageTarget.ts

```diff
@@ -4,15 +4,15 @@
  */
 
 import Model, {type ModelElement, type ModelProperties} from 'gmp/models/model';
-import {parseCsv, type YesNo, parseBoolean} from 'gmp/parser';
+import {parseCsv, parseBoolean} from 'gmp/parser';
 import {isDefined} from 'gmp/utils/identity';
 import {isEmpty} from 'gmp/utils/string';
 
 interface OciImageTargetElement extends ModelElement {
   image_references?: string;
   credential?: ModelElement;
-  reverse_lookup_only?: YesNo;
-  reverse_lookup_unify?: YesNo;
+  reverse_lookup_only?: boolean;
+  reverse_lookup_unify?: boolean;
```

Comment:
The data from the response aka XML Element is still `0` or `1`. Therefore it's of `YesNo` type.
```suggestion
  reverse_lookup_only?: YesNo;
  reverse_lookup_unify?: YesNo;
```